src/components/draggable-table-tags.tsx

```diff
@@ -103,15 +109,26 @@ export const DraggableTableHeader: React.FC<PropsWithChildren<DraggagleTableHead
           onMouseLeave={() => setShowDropdownIcon(false)}
           onClick={handleShowHeaderMenu}
         >
-          <div className={css.thChildContainer}>
-            <div>{children}</div>
-            {showDropdownIcon &&
-              <div className={css.dropdownIcon}>
-                <DropdownIcon
-                  onClick={handleShowHeaderMenu}
-                  className={css.dropdownIcon}
-                />
-              </div>
+          <div className={`${css.thChildContainer} ${isParent ? css.isParent : ""}`}>
+            <button
+              onMouseEnter={() => setShowDropdownIcon(true)}
+              onMouseLeave={() => setShowDropdownIcon(false)}
+              onClick={handleShowHeaderMenu}
+            >
+              {attrId && editableHasFocus
+                ? <EditableTableHeader
+                    attrId={attrId}
+                    collectionName={String(children)}
+                    collectionId={collectionId}
+                    hasFocus={editableHasFocus}
+                    renameAttribute={renameAttribute}
+                  />
+                : children}
+            </button>
+            {
+              <button className={css.dropdownIcon} onClick={handleShowHeaderMenu}>
+                {showDropdownIcon && <DropdownIcon className={css.dropdownIcon} />}
+              </button>
             }
```

Comment:
The `{ ... }` seems unnecessary here but maybe I'm missing something.

Reply:
You're right -- not necessary. I'll remove.